boards/adafruit-clue/reset.c

```diff
@@ -21,6 +21,7 @@
 
 #define USB_H_USER_IS_RIOT_INTERNAL
 
+#include "cpu.h"
```

Comment:
Why is this needed now ?

Reply:
The definition for the `NRF_POWER` register was previously pulled in through `usb_board_reset.h` -> `usb/usbus/cdc/acm.h` -> `usb.h` -> `board.h` -> `cpu.h`

Reply:
Ok, I see. Makes sense.